### ISSUE TITLE
Including source js/sass in npm package

### ISSUE THREAD

**Issue by lhokktyn** [hmrc/hmrc-frontend#96]
Would you consider including the `src/` directory in your published npm package, or perhaps providing it as a separate `html-frontend-src` package?

For performance reasons, we would like to compile the components ourselves, and together with our custom sources, roll them all up into single js/css files for serving to the browser.

At the moment we are including the source files direct from Github as an interim solution, which works fine, but it would be great to have these sources available alongside the views, etc to make it simpler to install within restricted environments.

**Comment by matthewmascord** [comment]
How are you consuming hmrc-frontend currently @lhokktyn? Do you have a link to your repository?

**Comment by mikeshawdev** [comment]
@matthewmascord we are currently pulling it directly from this git repository via `git+https://github.com/hmrc/hmrc-frontend.git#v1.30.0` in our `package.json` file. We can't link to our repository as it is private within our GitLab

**Comment by lhokktyn** [comment]
Back in the room ...

@matthewmascord here's an extract from our package.json:
```json
  ...
  "devDependencies": {
    "hmrc-frontend-src": "git+https://github.com/hmrc/hmrc-frontend.git#v1.36.0"
  }
  ...
```
And we're pulling specific components into our source, as so:
```javascript
/* JS */
import TimeoutDialog from 'hmrc-frontend-src/src/components/timeout-dialog/timeout-dialog';
```
```css
/* CSS */
@import "hmrc-frontend-src/src/components/timeout-dialog/timeout-dialog";
```
And compiled alongside our other stuff using rollup config:
```javascript
{
  ...
  babel({
      include: ['node_modules/hmrc-frontend-src/**/*'],
  }),
  ...
}
```

The aims here are primarily performance (delivering small/fewer assets to the browser).

**Comment by matthewmascord** [comment]
Hi @lhokktyn, one option might be for us to simply not roll up the individual Javascript modules (except for all.js) - and just provide them as original source instead. Would that work for you? I would have to check with the team about whether that is something we'd want to do.

**Comment by lhokktyn** [comment]
That would be pretty perfect @matthewmascord 👍

**Comment by matthewmascord** [comment]
Hi @lhokktyn - this has been implemented in v2.0.0: https://www.npmjs.com/package/hmrc-frontend

**Comment by lhokktyn** [comment]
Thanks for the quick turnaround @matthewmascord, much appreciated.
